ipatests/test_webui/test_subid.py

```diff
@@ -125,3 +132,20 @@ def test_user_subid_add(self):
         self.assert_no_error_dialog()
         after_count = self.get_rows()
         assert len(before_count) < len(after_count)
+
+    @screenshot
+    def test_subid_range_deletion_not_allowed(self):
+        """
+        Test to check that subid range delete is not
+        allowed from WebUI i.e Delete button is not available.
+        """
+        self.init_app()
+        self.navigate_to_entity('subid', facet='search')
+        self.facet_button_click('add')
+        self.select_combobox('ipaowner', 'admin')
+        self.dialog_button_click('add')
+        self.wait(0.3)
+        self.assert_no_error_dialog()
+        self.get_field_checked('ipauniqueid')
+        with pytest.raises(NoSuchElementException):
+            self.facet_button_click('delete')
```

Comment:
Based on the other pages in the UI, the name of the delete button is `remove`, not `delete` (`facet_button_click` matches `name` atrribute). 
Also, since we are trying to prove non-existence of a button, I would list all the buttons and ensure there are only `Add` and `Refresh`.
Something like (apart from what is above)
```
buttons_s = 'div.control-buttons button'
facet_buttons = self.find(buttons_s, By.CSS_SELECTOR, many=True, strict=True)
assert len(facet_buttons) == 2

assert facet_buttons[0].get_attribute('name') in ['add', 'refresh']
assert facet_buttons[1].get_attribute('name') in ['add', 'refresh']
```

Don't forget to import 
`from selenium.webdriver.common.by import By
`